azure-pipelines.vrt-baseline.yml

```diff
@@ -29,15 +29,25 @@ jobs:
           vrTestPackageName: '@fluentui/vr-tests-web-components'
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31997&commitId=1459fe1ca023dc80c098442c7b428c9471e71ffb&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.basic - Dark Mode.normal.chromium.png|0| Removed|
|Avatar Converged.basic - RTL.normal.chromium.png|0| Removed|

</details>


  <details><summary><b>Card Converged</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Card Converged.orientation.chromium.png|0| Removed|

</details>


  <details><summary><b>Card Converged - Interactive</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Card Converged - Interactive.appearance interactive - Outline - High Contrast.click.chromium.png|0| Added|
|Card Converged - Interactive.appearance interactive - Outline - High Contrast.hover.chromium.png|0| Added|
|Card Converged - Interactive.appearance interactive - Outline - High Contrast.normal.chromium.png|0| Added|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>